### ISSUE TITLE
[Bug]: uie-x f1只能达到0.55，并且labelstudio.py转换脚本生成的数据集bboxes生成错误

### ISSUE THREAD

**Issue by hehuang139** [PaddlePaddle/PaddleNLP#3822]
### 软件环境

```Markdown
- paddle-bfloat                  0.1.7
- paddle2onnx                    1.0.1
- paddlefsl                      1.1.0
- paddlenlp                      2.4.1.dev0  /home/hehuang/dev/git/uie-x
- paddleocr                      2.6.0.1
- paddlepaddle-gpu               2.4.0rc0
```

### 重复问题

- [X] I have searched the existing issues

### 错误描述

```Markdown
1. label_studio.py训练集创建bug，未生成bboxes
2. f1无法达到效果
3. ocr解析结果未拆分bbox？
```

### 稳定复现步骤 & 代码

1. 下载标记
wget https://paddlenlp.bj.bcebos.com/datasets/tax.tar.gz
tar -zxvf tax.tar.gz
mv tax data
rm tax.tar.gz

2. 通过label_studio.py 转换
python ../label_studio.py \
    --label_studio_file ./data/label_studio.json \
    --save_dir ./data \
    --splits 0.8 0.2 0\
    --task_type ext
发现生成的训练集bboxes字段为boxes，导致utils.py解析时无法解析到bbox信息，则认为是一般的text，不作为doc处理，这样训练确实可以得到f1 >0.95以上

错误代码1为：
paddlenlp.utils.convertor.Convertor#process_image_tag:189
items['boxes'] = boxes  --> 应该改为items['bboxes'] = boxes 

其他错误代码：
还有多处
items['boxes'] = boxes  --> 应该改为items['bboxes'] = boxes 

3. 修改bboxes正确后，，跑document/fineture.py，这是终于是layout doc了，但是F1只有0.55
[2022-11-19 07:06:34,554] [    INFO] - ***** train metrics *****
[2022-11-19 07:06:34,554] [    INFO] -   epoch                    =       10.0
[2022-11-19 07:06:34,554] [    INFO] -   train_loss               =     0.0012
[2022-11-19 07:06:34,554] [    INFO] -   train_runtime            = 0:27:47.64
[2022-11-19 07:06:34,554] [    INFO] -   train_samples_per_second =      3.598
[2022-11-19 07:06:34,554] [    INFO] -   train_steps_per_second   =       0.45
[2022-11-19 07:06:34,563] [    INFO] - ***** Running Evaluation *****
[2022-11-19 07:06:34,563] [    INFO] -   Num examples = 80
[2022-11-19 07:06:34,563] [    INFO] -   Total prediction steps = 10
[2022-11-19 07:06:34,563] [    INFO] -   Pre device batch size = 8
[2022-11-19 07:06:34,563] [    INFO] -   Total Batch size = 8
100%|██████████| 10/10 [00:20<00:00,  2.01s/it]
[2022-11-19 07:06:56,886] [    INFO] - ***** eval metrics *****
[2022-11-19 07:06:56,886] [    INFO] -   epoch                   =       10.0
[2022-11-19 07:06:56,886] [    INFO] -   eval_f1                 =     0.5556
[2022-11-19 07:06:56,887] [    INFO] -   eval_loss               =     0.0016
[2022-11-19 07:06:56,887] [    INFO] -   eval_precision          =      0.625
[2022-11-19 07:06:56,887] [    INFO] -   eval_recall             =        0.5
[2022-11-19 07:06:56,887] [    INFO] -   eval_runtime            = 0:00:22.32
[2022-11-19 07:06:56,887] [    INFO] -   eval_samples_per_second =      3.584
[2022-11-19 07:06:56,887] [    INFO] -   eval_steps_per_second   =      0.448
[2022-11-19 07:06:56,889] [    INFO] - Exporting inference model to ./checkpoint/model_best_no1/model

以下我的超参，
--device
gpu
--logging_steps
5
--save_steps
50
--eval_steps
50
--seed
42
--model_name_or_path
uie-x-base
--output_dir
./checkpoint/model_best_no1
--train_path
data/train.txt
--dev_path
data/dev.txt
--max_seq_len
512
--per_device_train_batch_size
8
--per_device_eval_batch_size
8
--num_train_epochs
10
--learning_rate
1e-5
--label_names
start_positions
end_positions
--do_train
--do_eval
--do_export
--export_model_dir
./checkpoint/model_best_no1
--disable_tqdm
False
--metric_for_best_model
eval_f1
--load_best_model_at_end
True
--save_total_limit
1
--overwrite_output_dir

4. 并且我发现一点，uie-x，并没有对ocr解析出来的text行，做单个字符bbox的拆分，但是奇怪的是，我手动改了拆分和不改拆分效果确实没有太大区别，F1，但是这个和layout的论文就不一样，这里是否就是可以不拆分bbox，还是代码的问题
paddlenlp.utils.convertor.Convertor#process_image_tag：181
        for segment in parsed_doc['layout']:
            box = doc_parser._normalize_box(segment[0], [img_w, img_h],
                                            [1000, 1000])
            text += segment[1]  
            boxes.extend([box] * len(segment[1]))   -- 这里没有拆分char？是否就是故意为之，因为看fineture，确实有一个判断同一行的text的input_ids的处理

**Comment by hehuang139** [comment]
@linjieccc 帮忙看下uie-x分支问题，效果并不好

**Comment by hehuang139** [comment]
以下是我验证的过程

1. 调整学习率5e-6,epoch为100，发现最后过拟合了，模型预测f1始终无法超过60%，说明确实50%是当前数据集的f1
2. 怀疑是否为字符未拆分bbox的原因，所以修改bbox，拆分字符bbox，但是f1也是没有变好，还是50%，和未拆分差不多
```
def cut_boxes(text, box):
    # 'x0': 296.4, 'x1': 298.893, 'top': 773.684, 'bottom': 782.684
    x_min, y_min = box[0], box[1]
    x_max, y_max = box[2], box[3]
    text_chars = list(text)
    char_num = 0
    for char in text_chars:
        if re.match("[^\x00-\xff]", char):
            char_num += 2
        else:
            char_num += 1
    width = x_max - x_min
    shift = x_min
    new_token_boxes, new_tokens = [], []
    for char in text_chars:
        if re.match("[^\x00-\xff]", char):
            tok_x_max = shift + width / char_num * 2
        else:
            tok_x_max = shift + width / char_num * 1
        tok_x_min = shift
        tok_y_min = y_min
        tok_y_max = y_max

        shift = tok_x_max
        new_token_boxes.append(
            [round(tok_x_min),
             round(tok_y_min), round(tok_x_max), round(tok_y_max)])
        new_tokens.append(char)

    return new_token_boxes
```
```
            text += segment[1]
            cutted_boxes = cut_boxes(text, box)
            boxes.extend(cutted_boxes)
```

**Comment by linjieccc** [comment]
Hi，@hehuang139 

感谢反馈问题以及对UIE-X相关工作的关注！目前这个分支的相关功能在持续开发中，可以等我们合入一个稳定版本到develop分支。

UIE中bbox的粒度是片段粒度，没有再做字的切分，doc_reader中的处理逻辑会分别对不同片段进行Tokenizer后合并在一起

**Comment by hehuang139** [comment]
@linjieccc applications.information_extraction.document.utils.reader:79/105 这两处拆分max_content_len的时候，boxes没有对应拆分.

**Comment by hehuang139** [comment]
@linjieccc 现在f1还是不行，能到70%，但是用微调的模型预测，完全不对，比直接用未微调的差非常多，请问f1有重新跑过吗，我现在有点怀疑是不是哪里有问题，
和taskflow的代码比较，summary_token_num = 4 有问题，但是也改了，重新跑还是效果不好，只能到71%，感觉基本没有代码差异了

**Comment by linjieccc** [comment]
@hehuang139 更新下这个分支的代码再试下 https://github.com/linjieccc/PaddleNLP/tree/uie-x

**Comment by hehuang139** [comment]
> @hehuang139 更新下这个分支的代码再试下 https://github.com/linjieccc/PaddleNLP/tree/uie-x

验证通过了
